simulator.py

```diff
@@ -91,6 +100,9 @@ def __init__(self, env, name, age, rng, infection_timestamp, household, workplac
         self.avg_working_hours = _draw_random_discreet_gaussian(AVG_WORKING_MINUTES, SCALE_WORKING_MINUTES, self.rng)
         self.scale_working_hours = _draw_random_discreet_gaussian(AVG_SCALE_WORKING_MINUTES, SCALE_SCALE_WORKING_MINUTES, self.rng)
 
+        self.avg_hospital_hours = _draw_random_discreet_gaussian(AVG_HOSPITAL_HOURS, SCALE_HOSPITAL_HOURS, self.rng)
+        self.scale_hospital_hours = _draw_random_discreet_gaussian(AVG_SCALE_HOSPITAL_HOURS, SCALE_SCALE_HOSPITAL_HOURS, self.rng)
+
```

Comment:
the number of hours a person will spend in the hospital should be proportional to his sickness. It should be really dependent on the severity of sickness

Reply:
fixed in admit function of hospital. it is now dependent on the symptoms